media/plg_media-action_smartcrop/css/smartcrop.css

```diff
@@ -1 +1,5 @@
-/* This is the CSS File for Smart Crop */
\ No newline at end of file
+/* This is the CSS File for Smart Crop */
+#attrib-smartcrop
+{
+    overflow-y: auto
+}
```

Comment:
Can you add a new line here?

Reply:
I have already added see [this](https://github.com/proman24/gsoc18_adaptive_images/blob/width_selection/media/plg_media-action_smartcrop/css/smartcrop.css).

Reply:
github  has some issues xD